client/state/login/selectors.js

```diff
@@ -142,7 +136,7 @@ export const isRequesting = ( state ) => {
  * @returns {?object}         Error for the request.
  */
 export const getRequestError = ( state ) => {
-	return get( state, 'login.requestError', null );
+	return state.login.requestError ?? null;
```

Comment:
Again, shouldn't need a fallback. Except if the `*_FAILURE` action objects had an empty `action.error` field...

Reply:
Agreed, done.